std/traits.d

```diff
@@ -1234,6 +1236,30 @@ template extractParameterStorageClassFlags(Attribs...)
     }();
 }
 
+///
+@safe unittest
+{
+    static void func(ref int ctx, out real result)
+    {
+    }
```

Comment:
FYI: It's really okay to do out everything in one line in such cases e.g. `func(...) {}